src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,33 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_output_element_type(const Napi::CallbackInfo& info) {
+    if(info.Length() != 1 || !info[0].IsNumber()) {
```

Comment:
Since [PR](https://github.com/openvinotoolkit/openvino/pull/24921) there is a new approach to parameter validation. In your case it will look like this:

```suggestion
    `if (ov::js::validate<Napi::Number>(info, allowed_signatures)) {}
 else OPENVINO_THROW('method_name', ov::js::get_parameters_error_msg(info, allowed_signatures))
```